src/urllib3/__init__.py

```diff
@@ -81,6 +81,7 @@
     "make_headers",
     "proxy_from_url",
     "request",
+    "BaseHTTPResponse"
```

Comment:
Can you add a comma to the end of this line to make the linter happy? (You can run `$ nox -s format` locally too).

Reply:
of course, one moment